README.md

```diff
@@ -170,6 +170,7 @@ Contributors
 - Ben Davies [Morfent] - Development
 - Ben Frengley [TalkTakesTime] - Development
 - Cody Thompson [Rising_Dusk] - Development
+- [HoeenHero] - Development
```

Comment:
He definitely goes in Major Contributors.

Have you asked him about crediting as real name?

Reply:
Also we should have jumbowhales in here.

Reply:
I did ask and he didn't want his real name on there at this moment.

Reply:
also mathfreak231